mi/dataset/driver/ctdav_nbosi/auv/ctdav_nbosi_auv_driver.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+"""
+@package mi.dataset.driver.ctdav_nbois.auv
+@file mi/dataset/driver/ctdav_nbois/auv/ctdav_nbois_auv_driver.py
+@author Rene Gelinas
+@brief Driver for the ctdav_nbois_auv instrument
+
+Release notes:
+
+Initial Release
+"""
+
+from mi.dataset.dataset_driver import SimpleDatasetDriver
+from mi.dataset.parser.ctdav_nbosi_auv import CtdavNbosiAuvParser
+from mi.core.versioning import version
+
+
+@version("1.0.0")
```

Comment:
1.0.0 is presumptuous (implies verification/validation over time) - initial versions are usually 0.1.0.

Reply:
Done.